app/main/views/services.py

```diff
@@ -37,7 +37,13 @@ def list_services():
 
     supplier_id = request.args.get('supplier_id')
 
-    services = Service.query.filter(Service.status == 'published')
+    services = Service.query
```

Comment:
Are we sure we want the default behaviour to be to return everything?  An alternative would be to return only published  by default, but take parameters to include expired/unpublished? Actually, I don't think there is any logic to exclude services belonging to expired Frameworks at the moment - should we get that in now too?